cli/cmd/install.go

```diff
@@ -231,6 +236,7 @@ func newInstallOptionsWithDefaults() *installOptions {
 			return id.String()
 		},
 
+		// TODO: rename to generateTLS or something
```

Comment:
Is this TODO for the PR or the future?

Reply:
For the future, `generateWebhookTLS` doesn't make sense for non-webhooks, but I'm trying to limit to size of this PR.